subworkflows/local/rank_variants.nf

```diff
@@ -27,15 +28,20 @@ workflow RANK_VARIANTS {
 
         GENMOD_COMPOUND(GENMOD_SCORE.out.vcf)
 
-        TABIX_BGZIPTABIX (GENMOD_COMPOUND.out.vcf)
+        BCFTOOLS_SORT(GENMOD_COMPOUND.out.vcf) // SV file needs to be sorted before indexing
```

Comment:
Where is the SV-file getting indexed?

Reply:
Index of this file isn't used by the downstream processes that's why I had removed it. But I don't mind putting it back in the pipeline if you guys think it'd be useful to have that feature 😄 

What do you guys think?

Reply:
I thought the whole point of running BCFTOOLS_SORT on the SV VCF-file was so that you could subsequently run TABIX_BGZIPTABIX? 🤔

Reply:
Smooth brain time! Sorry for the confusion, Anders. I just noticed that this PR is missing a commit (was working on a different machine and I forgot to push it). I had removed index from rank_variants step, and added it downstream, after filter_vep because the output from latter contains all the clinically relevant variants and is therefore more important for a user.

Reply:
Ah, ok. No worries. Let me know if you want a review of the PR and when it is ready for review. Cheers.

Reply:
More I think about this, I am starting to see how this can sometimes confuse users as well. So I think I am going to have the pipeline generate indices for both the rank_variant and filter_vep vcfs. Have a meeting now, but I will push the updates later today.